packages/cluster-ui/src/transactionsPage/transactionsPageConnected.tsx

```diff
@@ -0,0 +1,35 @@
+import { connect } from "react-redux";
+import { RouteComponentProps, withRouter } from "react-router-dom";
+import { Dispatch } from "redux";
+
+import { AppState } from "src/store";
+import { actions as statementActions } from "src/store/statements";
+import { actions as resetSQLStatsActions } from "src/store/sqlStats";
+import { TransactionsPage } from "./transactionsPage";
+import {
+  TransactionsPageProps,
+  TransactionsPageDispatchProps,
+} from "./transactionsPage";
+import {
+  selectTransactionsData,
+  selectTransactionsLastError,
+} from "./transactionsPage.selectors";
+import { nodeRegionsByIDSelector } from "../store/nodes";
+
+export const TransactionsPageConnected = withRouter(
+  connect<
+    TransactionsPageProps,
+    TransactionsPageDispatchProps,
+    RouteComponentProps
+  >(
+    (state: AppState) => ({
+      data: selectTransactionsData(state),
```

Comment:
@elkmaster @koorosh I'm getting a TypeScript error here, 

```
(alias) selectTransactionsData(state: AppState): any[] | cockroach.server.serverpb.StatementsResponse
import selectTransactionsData
No overload matches this call.
  The last overload gave the following error.
    Argument of type '(state: AppState) => { data: any[] | cockroach.server.serverpb.StatementsResponse; nodeRegions: { [key: string]: string; }; error: Error; }' is not assignable to parameter of type 'MapStateToPropsParam<TransactionsPageProps, RouteComponentProps<{}, StaticContext, unknown>, DefaultRootState>'.
      Type '(state: AppState) => { data: any[] | cockroach.server.serverpb.StatementsResponse; nodeRegions: { [key: string]: string; }; error: Error; }' is not assignable to type 'MapStateToPropsFactory<TransactionsPageProps, RouteComponentProps<{}, StaticContext, unknown>, DefaultRootState>'.
        Type '{ data: any[] | cockroach.server.serverpb.StatementsResponse; nodeRegions: { [key: string]: string; }; error: Error; }' is not assignable to type 'MapStateToProps<TransactionsPageProps, RouteComponentProps<{}, StaticContext, unknown>, DefaultRootState>'.
          Type '{ data: any[] | StatementsResponse; nodeRegions: { [key: string]: string; }; error: Error; }' provides no match for the signature '(state: DefaultRootState, ownProps: RouteComponentProps<{}, StaticContext, unknown>): TransactionsPageProps'.ts(2769)
index.d.ts(213, 5): The last overload is declared here.
```

Not convinced I have this right yet, but I don't see anything missing from the Transactions Page props.